files/fielddata/mapmatrix/map_matrix.mk

```diff
@@ -0,0 +1,7 @@
+FIELDDATA_MAPMATRIX_MAP_MATRIX_DIR := files/fielddata/mapmatrix/map_matrix
+MATRIX_FILES := $(sort $(wildcard $(FIELDDATA_MAPMATRIX_MAP_MATRIX_DIR)/*.bin))
+
+$(FIELDDATA_MAPMATRIX_MAP_MATRIX_DIR).narc: %.narc: $(MATRIX_FILES)
```

Comment:
Since all the dependencies have the `.bin` extension, this can fall back to the default prerequisite spec for the `%.narc` pattern

Reply:
(If the bin are built, the explicit dependency would become required)